src/xlsx.js

```diff
@@ -0,0 +1,99 @@
+export class ExcelWorkbook {
+  constructor(workbook) {
+    Object.defineProperty(this, "_", {value: workbook});
+  }
+  sheetNames() {
+    return this._.worksheets.map((sheet) => sheet.name);
+  }
```

Comment:
Can we make this an enumerable property rather than a method, so that it’s visible in the inspector and feels more consistent with ZipArchive?

https://github.com/observablehq/stdlib/blob/466685ddb0eb60c2192b550b5bc57d5ed0018baa/src/fileAttachment.js#L100-L110

Reply:
Why writable?